backend/s3/s3.go

```diff
@@ -1390,7 +1415,19 @@ func s3Connection(opt *Options) (*s3.S3, *session.Session, error) {
 		SessionToken:    opt.SessionToken,
 	}
 
-	lowTimeoutClient := &http.Client{Timeout: 1 * time.Second} // low timeout to ec2 metadata service
+	defaultClient := getClient(opt)
+	defaultClient.Timeout = 1 * time.Second
+	lowTimeoutClient := defaultClient
+
+	//defaultTransport := new(http.Transport)
```

Comment:
This was the earlier default i.e create a new client instead of using the one given by fshttp. 

Do you want me to revert to that behaviour and uncomment the following code or the new code is fine and I can delete the commented part?

Reply:
I don't think you need to touch the lowTimeoutClient - it is only used for communicating with the local host server to get credentials - it won't use HTTP/2 for that so we don't need to disable it.